woocommerce/woocommerce-gutenberg-products-block
# Update Features Flags and Experimental Interfaces doc with Collection Filters and its inner blocks

## What

Update Features Flags and Experimental Interfaces doc with Collection Filters and its inner blocks.

## Why

Keep the doc up to date and make the tracking of flags easier.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Check whether newly added links redirect to the flags
2. Verify MD file is formatted correctly


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
